test/Microsoft.Azure.WebJobs.Host.UnitTests/ConverterManagerTests.cs

```diff
@@ -40,11 +40,37 @@ public void ExactMatchOverride()
             cm.AddConverter<string, string>(x => "*" + x + "*");
 
             var func = cm.GetConverter<string, string, Attribute>();
-                        
+
             var x1 = func("x", null, Context);
             Assert.Equal("*x*", x1);
         }
 
+        public T TestDefaultConverter<F, T>(F from, T to)
+        {
+            var cm = new ConverterManager();
+            var converter = cm.GetConverter<F, T, Attribute>();
+            Assert.NotNull(converter);
+            var result = converter.Invoke(from, null, null);
+            Assert.Equal(to, result);
+            return result;
+        }
+
+        [Fact]
+        public void ByteToString_DefaultConverter()
+        {
+            var s = "ab";
+            TestDefaultConverter(Encoding.UTF8.GetBytes(s), s);
+        }
+
+        [Fact]
+        public void IEnumerableToJArray_DefaultConverter()
```

Comment:
>IEnumerableToJArray_DefaultConverter [](start = 20, length = 36)

Can you include another test to demonstrate chaining? 

Ie, add a converter from Widget --> IEnumerable<JObject> 
and then verify we can do a get (Widget, JArray)?